### PR TITLE
Azure form changes for new designs

### PR DESCRIPTION
**Pull request by mikewrosey**
## Description
Azure form changes for new designs
Success modal for multiple discoveries

As a follow up to this, there will be tags + new endpoint change

## Jira link(s)
- https://issues.opennms.org/browse/HS-1021
- Parent: https://issues.opennms.org/browse/HS-955

screenshot-localhost_3009-2023 02 15-16_30_27

screenshot-localhost_3009-2023 02 15-16_25_20

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

File: ui/src/components/Discovery/AzureForm.vue [hunk 1]
```diff
@@ -1,60 +1,136 @@
...
+    .name, .locations, .tags {
+      width: calc(50% - var(variables.$spacing-s))
+    }
+    .row {
+      display: flex;
+      gap: var(variables.$spacing-l);
+      .column {
+        flex: 1;
+      }
+    }
```
Comment (chinh-cle) [inline ui/src/components/Discovery/AzureForm.vue:133]: should all discovery filename starting with `Discovery...` ?

File: ui/src/components/Discovery/DiscoverySuccessModal.vue [hunk 2]
```diff
@@ -0,0 +1,132 @@
+<template>
+  <PrimaryModal :visible="isVisible" title="''" hideTitle>
```
Comment (chinh-cle) [inline ui/src/components/Discovery/DiscoverySuccessModal.vue:2]: is '' in title attribute required?

**Review by chinh-cle — APPROVED** [review]

File: ui/src/components/Discovery/AzureForm.vue (hunk 1 above)
Comment (mikewrosey) [reply ui/src/components/Discovery/AzureForm.vue]: Yup, it should be. Renamed

File: ui/src/components/Discovery/DiscoverySuccessModal.vue (hunk 2 above)
Comment (mikewrosey) [reply ui/src/components/Discovery/DiscoverySuccessModal.vue]: It was, but there's a better way to handle it in the primary modal comp itself. Title removed from here now.

**Pull request merged**